ngraph/python/src/pyngraph/frontend_manager.cpp

```diff
@@ -101,3 +102,38 @@ void regclass_pyngraph_FEC(py::module m)
            const ngraph::frontend::FrontEndCapabilities& b) { return a == b; },
         py::is_operator());
 }
+
+void regclass_pyngraph_ErrorCode(py::module m)
```

Comment:
I have compilation error, did you forget to include frontmanager_exceptions.hpp?

Reply:
fixed